### PR TITLE
{bio}[foss/2017b,intel/2017b] RELION v3.0.4

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/2f24c199c0601f9444b15a5cf5c0ebd2 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/2cf5913f0a2f4cc343f86ca7c60b1d23 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @vanzod!

File: easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ('X11', '20171023'),
+    ('FLTK', '1.3.4'),
+    ('LibTIFF', '4.0.9'),
+    ('tbb', '2018_U5'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread'  -DMPI_INCLUDE_PATH=$EBROOTOPENMPI/include "
+configopts += "-DMPI_C_COMPILER=$EBROOTOPENMPI/bin/mpicc -DMPI_CXX_COMPILER=$EBROOTOPENMPI/bin/mpicxx "
+configopts += "-DCUDA=OFF -DCudaTexture=OFF "
+configopts += "-DALTCPU=ON -DMKLFFT=ON -DFORCE_OWN_TBB=OFF "
```
Comment (akesandgren) [inline easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb]: You probably shouldn't have MKLFFT=ON in the foss build.

Comment (migueldiascosta) [reply easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb]: oops... @vanzod, do you want to submit a bugfix PR?

Comment (boegel) [reply easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb]: Yeah, would be nice to fix that.

Why does it even work though? :-/

Comment (vanzod) [reply easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb]: Fixed in #8043

Comment (boegel) [reply easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb]: Bugfix done in #8043, thanks @vanzod!

Comment (akesandgren) [reply easybuild/easyconfigs/r/RELION/RELION-3.0.4-foss-2017b.eb]: It likely detects that there ARE no MKLFFT available and does something else?

**Comment by akesandgren** [comment]
For a variation on this see #8157

**Pull request merged**